functions/tab.xfce4_terminal.fish

```diff
@@ -0,0 +1,3 @@
+function tab.xfce4_terminal -a cdto cmd
+  xfce4-terminal --tab --working-directory "$cdto" -e "fish -c '$cmd; exec fish'"
```

Comment:
Oh, clever. 

Do we need to specify "interactive" or "login", or are the shells set up properly?

Will there be any escaping issues?